### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Dima-Semenov**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -7,4 +7,20 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  it('should return correct value if catAge && dogAge < 15', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  it('should return correct value if 15 <= catAge && dogAge <= 28', () => {
```
Comment (Oleksandra-Pasenchenko) [inline src/getHumanAge.test.js:16]: I would recommend adding test for 29 for dogAge - it's a boundary value for this interval

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -7,4 +7,20 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  it('should return correct value if catAge && dogAge < 15', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
```
Comment (Oleksandra-Pasenchenko) [inline src/getHumanAge.test.js:13]: Better write each 'expect' separately, so that you check only one situation in one test

**Review by Oleksandra-Pasenchenko — CHANGES_REQUESTED** [review]

**Review by Oleksandra-Pasenchenko — CHANGES_REQUESTED** [review]
Fix comments, please

**Review by Oleksandra-Pasenchenko — APPROVED** [review]
Good job!
